css/style.css

```
@@ -15,5 +15,147 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+ 
+ nav  {
+    display:flex;
+    justify-content: flex-end;
```

```suggestion
    justify-content: space-around;
```
We can change this to `space-around`, and delete the `#logo`'s `margin-right:39em;` below on line 34, for a better way to put space between the logo and the menu links